dev/tidb-in-kubernetes/maintain/backup-and-restore.md

```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
 Incremental backup uses [TiDB Binlog](/dev/reference/tidb-binlog-overview.md) to collect binlog data from TiDB and provide real-time backup and replication to downstream platforms.
 
 For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/dev/tidb-in-kubernetes/maintain/tidb-binlog.md).
+
+### Scale Pump in
+
+To decrease the capability of Pump, make Pump nodes offline and then run the `helm upgrade` command to delete Pump pods.
```

Comment:
```suggestion
To scale Pump in, you decrease the number of Pump nodes by taking the nodes offline one at a time and running the `helm upgrade` command to delete the corresponding Pump Pod. The steps are as follows:
```
"Pod" needs to be capitalized. ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/